common/tee.edl

```diff
@@ -43,6 +44,9 @@ enclave
             [out, size=key_info_size] void* key_info,
             size_t key_info_size,
             [out] size_t* key_info_size_out);
+
+        public oe_result_t oe_handle_init_switchless(
+            [in] oe_switchless_call_manager_t* manager);
```

Comment:
Passing it by pointer would imply count=1, and the struct would be copied into the enclave.
Consider either passing by value or avoiding foreign structures and pass necessary values are parameters.

Reply:
Resolved.